src/groups/mqb/mqbblp/mqbblp_domain.cpp

```diff
@@ -925,7 +925,7 @@ void Domain::removeDomainReset()
     bslmt::LockGuard<bslmt::Mutex> guard(&d_mutex);  // LOCK
 
     d_state            = e_PREREMOVE;
-    d_teardownRemoveCb = nullptr;
+    d_teardownRemoveCb = bsl::nullptr_t();
```

Comment:
I find the use of `nullptr_t`'s constructor a little controversial, I feel as if we should really only be using preexisting representations of a nullptr (e.g. NULL, nullptr) but I suppose this works?

Reply:
> I find the use of `nullptr_t`'s constructor a little controversial, I feel as if we should really only be using preexisting representations of a nullptr (e.g. NULL, nullptr) but I suppose this works?

Pure `nullptr` doesn't work on Solaris, `NULL` or `bsl::nullptr_t()` work, but `bsl::nullptr_t()` is more restrictive. We typically used `bsl::nullptr_t()` to reset callbacks (`bsl::function`)